src/ResourceManager/Consumption/Commands.Consumption/help/Get-AzureRmConsumptionUsageDetail.md

```diff
@@ -166,12 +213,42 @@ Accept wildcard characters: False
 ```
 
 ### -StartDate
-The start date (in UTC) of the usages.
+The start date (in UTC) of the usages to filter.
 
 ```yaml
 Type: DateTime
 Parameter Sets: (All)
-Aliases: 
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Tags
```

Comment:
Be sure to rengenerate the file after changing Tags to Tag to reflect the change.

Reply:
.md file is regenerated accordingly.

Reply:
Looks like it is still tags

Reply:
Change it to tag